src/legacy/server/config/schema.js

```diff
@@ -193,7 +193,7 @@ export default () =>
         .default('localhost'),
       watchPrebuild: Joi.boolean().default(false),
       watchProxyTimeout: Joi.number().default(10 * 60000),
-      useBundleCache: Joi.boolean().default(Joi.ref('$prod')),
+      useBundleCache: Joi.boolean().default(!!process.env.CODE_COVERAGE ? true : Joi.ref('$prod')),
```

Comment:
The problem is somehow without setting `useBundleCache` in a schema, I'm getting[ flaky tests](https://kibana-ci.elastic.co/job/elastic+kibana+code-coverage/142/) and maybe it is because we don't make a build, but run tests from the source. This change was also made in discussion with @spalger. If I remove it, tests are started to fail for code coverage runs.